benchmark/browser/scenarios/grid-system/index.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import Grid from '@mui/system/Grid';
+import Grid from '@mui/system/Unstable_Grid';
```

Comment:
It might have been clearer with:

```suggestion
import Grid from '@mui/system/Unstable_Grid2';
```

under the idea that the code namespace is unique inside MUI.